Packs/CommonPlaybooks/Playbooks/playbook-Get_host_forensics_-_Generic.yml

```diff
@@ -261,8 +263,36 @@ inputs:
   value: {}
   required: false
   description: Provide the machine IDs of the systems you want to retrieve.
-  playbookInputQuery: null
-outputs: []
+  playbookInputQuery:
+outputs:
+- contextPath: MicrosoftATP
```

Comment:
are they any more integrations in the playbook? Besides Microsoft APT and illusive? That we might want to add their outputs as well?

Reply:
Currently only these two :)